ambuda/models/auth.py

```diff
@@ -0,0 +1,90 @@
+from flask_login import UserMixin
+from enum import Enum
+from datetime import datetime
+from sqlalchemy import (
+    Column,
+    DateTime,
+    Integer,
+    String,
+)
+from sqlalchemy.orm import relationship
+from werkzeug.security import generate_password_hash, check_password_hash
+
+from ambuda.models.base import Base, foreign_key, pk
+
+
+class SiteRole(Enum):
+    PROOFREADER = "Allow proofreading of texts prior."
```

Comment:
I like the idea, but it feels odd to me to have a description be on the RHS of an ENUM, since the RHS is basically the ground truth that we use and test in our logic. Something like:

```
class SiteRole(str, Enum):
    #: Can mark pages as yellow
    P1 = 'p1'
    #: Can mark pages as green
    P2 = 'p2'
    #: Administrator rights
    ADMIN = 'admin'
```

feels more intuitive. I think descriptions are useful if the enum values here are more user-facing, but here we're mainly using them for permissions tracking -- I think just `id` and `name` are necessary for our needs, and we can keep the descriptions in comments.